lib/mpl_toolkits/mplot3d/axes3d.py

```diff
@@ -2707,6 +2708,299 @@ def calc_arrow(uvw, angle=15):
 
     quiver3D = quiver
 
+    def errorbar(self, x, y, z, zerr=None, yerr=None, xerr=None, fmt='',
+                 barsabove=False, errorevery=1, ecolor=None, elinewidth=None,
+                 capsize=None, capthick=None, xlolims=False, xuplims=False,
+                 ylolims=False, yuplims=False, zlolims=False, zuplims=False,
+                 **kwargs):
+        """
+        Draws error bars on an Axis3D instance.
+
+        Parameters
+        ----------
+        x : scalar
```

Comment:
I may be missing something here, but should `errorbar` be able to take an array of data points to plot more than one bar at a time?